packages/gatsby/src/redux/actions.js

```diff
@@ -1099,13 +1099,18 @@ actions.createRedirect = ({
     pathPrefix = store.getState().config.pathPrefix
   }
 
+  // Check if URL is an external link
+  // and if it is, don't add pathPrefix
+  const isExternalUrl = toPath.startsWith(`https://` || `http://`)
```

Comment:
> or
```js
const isExternalUrl = /^https?/.test(toPath)
```

@DSchau shouldn't this be:
```js
const isExternalUrl = /^https?:/.test(toPath)
```
(added colon) in order to prevent edge cases with paths beginning with `http` not as a protocol?

Reply:
Yep! Good catch.